src/core/index.ts

```diff
@@ -1,4 +1,5 @@
-import type { NitroConfig } from "nitropack/types";
+import type { NitroConfig, NitroRouteConfig } from "nitropack/types";
+import type { NitroOpenAPIConfig } from "../types/openapi";
```

Comment:
Can we use nitropack/types entry to import this?

Reply:
It wasn't exported, but if you're happy for it to be, then yes!

Reply:
At least for v2 lets do it to keep subpath boundaries clear 👍